li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaFederatedConsumerImpl.java

```diff
@@ -113,6 +119,33 @@ public ClusterDescriptor getCluster() {
   // Number of config reload operations executed
   private volatile int _numConfigReloads;
 
+  private class PartitionsByClusterResult {
```

Comment:
some javadoc on what this is - the result of looking up the location of topics across multiple clusters in a group